ncm-openstack/src/main/pan/components/openstack/sudo.pan

```diff
@@ -0,0 +1,47 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+# ${build-info}
+
+@documentation{
+The goal of the root wrapper is to allow a service-specific unprivileged 
+user to run a number of actions as the root user, in the safest manner possible.
+Some OpenStack services (nova, neutron,..) must execute this wrapper as root.
+More info:
+https://wiki.openstack.org/wiki/Rootwrap
+}
+
+unique template components/${project.artifactId}/sudo;
+
+include 'components/sudo/config';
+
+"/software/components/sudo/privilege_lines" = {
+    sudolist = list(
```

Comment:
make this a dict, with 
```pan
sudos = dict(
  'nova', list(...),
  'neutron', list(...),
);
```

and loop over it per user, instead of copy/pasting blocks of pan code